elixir/apps/domain/lib/domain/accounts/account/changeset.ex

```diff
@@ -2,22 +2,61 @@ defmodule Domain.Accounts.Account.Changeset do
   use Domain, :changeset
   alias Domain.Accounts.Account
 
-  def create_changeset(attrs) do
-    %Account{}
+  def changeset(account, attrs) do
+    account
     |> cast(attrs, [:name, :slug])
     |> validate_required([:name])
-    |> validate_length(:name, min: 1, max: 255)
+    |> validate_name()
+    |> trim_name_whitespace()
```

Comment:
We have a helper for that: https://github.com/firezone/firezone/blob/andrew/actors-ui/elixir/apps/domain/lib/domain/validator.ex#L395

Reply:
Oh, awesome!  I need to look through that `validator.ex` file closer.  I'll get this updated.

Reply:
Updated this to use the function in the `validator.ex` file